### PR TITLE
[DMP-652] Added loading state to DetailProductView

### PR DESCRIPTION
**Pull request by luisdeacero**
## What is done

[[DMP-652] [Frontend] Agregar estado de loading en Detalle de producto](https://deacero.atlassian.net/browse/DMP-652)

## How is done

1. If product.groupedProduct has a value
2. Show the page as normal
3. If product.groupedProduct has no value
4. Show an html with divs and p with styles added to mock an empty page

## How to test

1. Run the project via `npm start`
2. Choose a product
3. You can view the page before it recieves a response with products info
4. Or go to ProductDetailView.tsx on line 269 and change product.groupedProduct to false
5. And you can view the loading state of the page indefinetly

## Evidence

Add images or screen  recordings on how the feature was working after and before the fix (if applies)

### CODE REVIEW

File: src/views/ProductDetailView/LoadingProduct/LoadingProduct.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                ></p>
+                <p
+                  className={`${styles["loading"]} ${styles["quantity-label-loading"]}`}
+                ></p>
+                <p
+                  className={`${styles["loading"]} ${styles["quantity-input-loading"]}`}
+                ></p>
+                <p
+                  className={`${styles["loading"]} ${styles["quantity-button-loading"]}`}
+                ></p>
```
Comment (markriverac) [inline src/views/ProductDetailView/LoadingProduct/LoadingProduct.tsx:70]: You could use a loop to render these as they are the same, so we avoid code duplication

File: src/views/ProductDetailView/LoadingProduct/LoadingProduct.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+              ></p>
+              <p
+                className={`${styles["loading"]} ${styles["summary-description-loading"]}`}
+              ></p>
+              <p
+                className={`${styles["loading"]} ${styles["summary-description-loading"]}`}
+              ></p>
+              <p
+                className={`${styles["loading"]} ${styles["summary-description-loading"]}`}
+              ></p>
```
Comment (markriverac) [inline src/views/ProductDetailView/LoadingProduct/LoadingProduct.tsx:90]: Same here, try to avoid code duplication

**Review by markriverac — CHANGES_REQUESTED** [review]
Thank you a lot for the refactoring. Just some small changes ;)

File: src/views/ProductDetailView/LoadingProduct/LoadingProduct.tsx (hunk 2 above)
Comment (luisdeacero) [reply src/views/ProductDetailView/LoadingProduct/LoadingProduct.tsx]: Done

File: src/views/ProductDetailView/LoadingProduct/LoadingProduct.tsx (hunk 1 above)
Comment (luisdeacero) [reply src/views/ProductDetailView/LoadingProduct/LoadingProduct.tsx:73]: They are not duplicates on the last assigned className, but I could use a numeric value on styles to force a loop. Should I go with something like that or leave it as it is?

Comment (markriverac) [reply src/views/ProductDetailView/LoadingProduct/LoadingProduct.tsx:73]: Oh, I didn't catch that. Okay, I'll approve

**Pull request merged**